lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -890,18 +888,13 @@
       }
 
       &.course-status-processing {
-        background-color: $gray-l5;
-        border: 0;
+
       }
 
       &.course-status-certnotavailable {
-        background-color: $gray-l5;
-        border: 0;
```

Comment:
If this is triggered, we want to make sure that the condition (cert not available) is not indicated by this color change alone.